### ISSUE TITLE
.Rproj is in a subdirectory of your git repo

### ISSUE THREAD

**Issue by ratnanil** [pampicloud/cma-week1#3]
You have created the RStudio Project as a subdirectory of the Git Repo. Is there a reason for this? I would recommend always creating the git repo and the RProject on the same hierarchical level.

**Comment by pampicloud** [comment]
Yeah. The reason was, that my RStudio freezed everytime if the RProject was on the same hierarchical level.
